vagrant/provisioning/roles/arkcase-app/templates/server.yml

```diff
@@ -116,3 +116,9 @@ calendar:
   :\"{{ calendar_purge_option }}\",\"daysClosed\":null}}"
 acm.hazelcast:
   log.folder: "{{ root_folder }}/log/arkcase"
+
+################### AWS Credentials (aws/credentials.properties) ####################
+## AccessKeyId and SecretAccessKey must be encrypted. ##
```

Comment:
Do we have to encrypt the access key and the secret access key ourselves?  If so where is the task to encrypt them?

Reply:
In would be easy to encrypt them. Just add the variable in "encrypt passwords" task and use it in the template like: "ENC({{ encrypted_aws_secret_access_key }})"